ci/docker/python-wheel-windows-vs2019-base.dockerfile

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# NOTE: You must update PYTHON_WHEEL_WINDOWS_IMAGE_REVISION in .env
+# when you update this file.
+
+# based on mcr.microsoft.com/windows/servercore:ltsc2019
+# contains choco and vs2019 preinstalled
+FROM abrarov/msvc-2019:2.11.0
```

Comment:
Can we use `python-wheel-windows-test-vs2019-base.dockerfile` as the base image?

Reply:
Why would we? That would make things more complicated.

Reply:
Because I felt that `python-wheel-windows-{,test-}vs2019-base.dockerfile` have many duplication. But I re-read them now, there are not so much duplication. Sorry.